src/include/sof/drivers/sai.h

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright 2019 NXP
+//
+// Author: Jerome Laclavere <jerome.laclavere@nxp.com>
+// Author: Guido Roncarolo <guido.roncarolo@nxp.com>
```

Comment:
C comments on headers. Assemblers may include these headers and not recognize C++ comments.

Reply:
ok

Reply:
FYI: Coding style convention is here. So, basically what @paulstelian97 said

https://elixir.bootlin.com/linux/v5.2-rc1/source/Documentation/process/license-rules.rst#L71

Reply:
ok